src/services/rollouts.js

```diff
@@ -540,4 +543,216 @@ module.exports = class RolloutsHelper {
 			return error
 		}
 	}
+	/**
+	 * rollout publish
+	 * @method
+	 * @name publish
+	 * @param {Integer} rolloutId - rollout id
+	 * @returns {JSON} - rollout publish response.
+	 */
+
+	static async publish(rolloutId, orgId, loggedInUserId) {
```

Comment:
@adithyadinesh0412  can u add this in validator and do basic checks

Reply:
This is a get call and accepting only Id from the DB. @priyanka-TL